### ISSUE TITLE
ONNX predict missing token_type_ids input

### ISSUE THREAD

**Issue by whr778** [ThilinaRajapakse/simpletransformers#1009]
**Describe the bug**
ONNX model predict is missing an input token_type_ids

**To Reproduce**
Save any model as onnx reopen the model as onnx call predict on the model
Additionally, the return values from predict return predictions for torch models and for onnx predictions and logits are returned... that wasn't expected... but easy to work around

**Expected behavior**
ner_model 1017-1033 should be 

        if self.args.onnx:
            model_inputs = self.tokenizer.batch_encode_plus(
                to_predict, return_tensors="pt", padding=True, truncation=True
            )

            for (input_ids, attention_mask, token_type_ids) in tqdm(zip(model_inputs["input_ids"],
                                                                        model_inputs["attention_mask"],
                                                                        model_inputs["token_type_ids"])):
                input_ids = input_ids.unsqueeze(0).detach().cpu().numpy()
                attention_mask = attention_mask.unsqueeze(0).detach().cpu().numpy()
                token_type_ids = token_type_ids.unsqueeze(0).detach().cpu().numpy()
                inputs_onnx = {"input_ids": input_ids, "attention_mask": attention_mask,
                               "token_type_ids": token_type_ids}

                # Run the model (None = get all the outputs)
                output = self.model.run(None, inputs_onnx)

**Screenshots**

**Desktop (please complete the following information):**
 - OS Linux ... ubuntu 20.04 and aws linux2

**Additional context**
Tried to push a branch and was going to submit a merge request... can I be made a contributor on this project?

**Comment by whr778** [comment]
A better example...
```
       if self.args.onnx:

            model_inputs = self.tokenizer.batch_encode_plus(
                to_predict, return_tensors="pt", padding=True, truncation=True
            )

            for (input_ids, attention_mask, token_type_ids) in tqdm(zip(model_inputs["input_ids"],
                                                                        model_inputs["attention_mask"],
                                                                        model_inputs["token_type_ids"])):
                if self.args.model_type not in ["roberta", "gpt2"]:
                    input_ids = input_ids.unsqueeze(0).detach().cpu().numpy()
                    attention_mask = attention_mask.unsqueeze(0).detach().cpu().numpy()
                    token_type_ids = token_type_ids.unsqueeze(0).detach().cpu().numpy()
                    inputs_onnx = {"input_ids": input_ids, "attention_mask": attention_mask,
                                   "token_type_ids": token_type_ids}
                else:
                    input_ids = input_ids.unsqueeze(0).detach().cpu().numpy()
                    attention_mask = attention_mask.unsqueeze(0).detach().cpu().numpy()
                    inputs_onnx = {"input_ids": input_ids, "attention_mask": attention_mask}

                # Run the model (None = get all the outputs)
                output = self.model.run(None, inputs_onnx)

                if preds is None:
                    preds = output[0]
                    out_input_ids = inputs_onnx["input_ids"]
                    out_attention_mask = inputs_onnx["attention_mask"]
                else:
                    preds = np.append(preds, output[0], axis=0)
                    out_input_ids = np.append(out_input_ids, inputs_onnx["input_ids"], axis=0)
                    out_attention_mask = np.append(out_attention_mask, inputs_onnx["attention_mask"], axis=0,)
            out_label_ids = np.zeros_like(out_input_ids)
            for index in range(len(out_label_ids)):
                out_label_ids[index][0] = -100
                out_label_ids[index][-1] = -100
        else:
                ...
```
